### PR TITLE
Remove unreachable code in SessionManager

### PR DESCRIPTION
**Pull request by Xerkus**
Traversable case is already covered in previous conditional block

### CODE REVIEW

File: src/SessionManager.php [hunk 1]
```diff
@@ -139,10 +139,6 @@ public function start($preserveStorage = false)
 
         session_start();
 
-        if ($oldSessionData instanceof Traversable) {
```
Comment (michalbundyra) [inline src/SessionManager.php:142]: If we remove this part do we then need line 12 in that file? (Importing of `Traversable`)

Comment (Xerkus) [reply src/SessionManager.php:142]: changed other Traversable to use import instead of FQCN

**Comment by michalbundyra** [comment]
I was digging it a bit before, and now I think it is not possible to have Traversable `$_SESSION` and not `StorageInterface`. I think this because we allow only `StorageInterface` as session storage, and we should check only this.
I know that before we check there for `Traversable` but I think it was wrong (it doesn't work correctly anyway) - so I think now we should remove traversable conditions at all.

**Comment by Xerkus** [comment]
It is superglobal, it can be set by something else and for that reason check makes sense.

**Comment by michalbundyra** [comment]
@Xerkus 
> It is superglobal, it can be set by something else and for that reason check makes sense.

Should we care about it if it was set by something else, and it's not compatible with the library?

I'm not sure this conversion to array for traversable, because it is not recursive, and not sure if it should be....

**Comment by Xerkus** [comment]
yay. way to go pecl, way to go...
```
$ pecl -q install mongodb
pecl/mongodb is already installed and is the same as the released version 1.3.3
install failed
```

**Comment by weierophinney** [comment]
Thanks, @Xerkus

**Pull request merged**